svc-api/main.go

```diff
@@ -66,20 +66,32 @@ func main() {
 				yes := strings.Contains(basicAuth, "Basic")
 				if yes {
 					spl := strings.Split(basicAuth, " ")
+					if len(spl) != 2 {
```

Comment:
check if this can be any other space character(tab ??)

Reply:
Basic auth will have only space between the basic keyword and encoded string, If anything apart from space comes in it will show an error message